custom-vu/src/main/kotlin/jces1209/vu/page/DcIssueNavigator.kt

```diff
@@ -0,0 +1,52 @@
+package jces1209.vu.page
+
+import jces1209.vu.wait
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.ui.ExpectedConditions.*
+import java.time.Duration
+import java.util.*
+
+
+class DcIssueNavigator(
+    driver: WebDriver
+): IssueNavigator {
+    private val webDriver = driver
+    private val page = FalliblePage.Builder(
+        driver,
+        presenceOfElementLocated(By.className("issue-list"))
+    )
+        .timeout(Duration.ofSeconds(30))
+        .build()
+
+    override fun waitForNavigator() {
+        page.waitForPageToLoad()
+    }
+
+    override fun getIssueElementFromList(): WebElement {
+        val elements = webDriver.wait(
+            presenceOfAllElementsLocatedBy(By.xpath("//*[@class ='issue-list']/*"))
```

Comment:
also, you could this locator extract to object field and reuse in other method - like FalliblePage
```suggestion
            presenceOfAllElementsLocatedBy(By.className("issue-list"))
```

Reply:
Here I'm getting child elements of //*[@class ='issue-list'] element. So left as is